docs/src/design.rst

```diff
@@ -4,9 +4,11 @@
 Design overview
 ===============
 
-libuv is cross-platform support library which was originally written for NodeJS. It's designed
+libuv is cross-platform support library which was originally written for `Node.js`_. It's designed
 around the event-driven asynchronous I/O model.
 
+.. _Node.js: http://nodejs.org
```

Comment:
HTTP? It's 2019!

Reply:
@saghul  Thanks for the catch. Changed it to https url.